jenkins/prepareOskar.ps1

```diff
@@ -1,4 +1,19 @@
-﻿$HDD = $(Split-Path -Qualifier $env:WORKSPACE)
+﻿Function proc($process,$argument)
+{
+    $p = Start-Process $process -ArgumentList $argument -NoNewWindow -PassThru
+    $h = $p.Handle
+    $p.WaitForExit()
```

Comment:
Can we use `-Wait` parameter in `Start-Process` instead of this?

Reply:
No because the -wait parameter is broken in Powershell